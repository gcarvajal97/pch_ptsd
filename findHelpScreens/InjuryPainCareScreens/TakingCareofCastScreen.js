import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import translate from '../../components/translateService';
import colors from '../../colors';

export default class TakingCareofCastScreen extends Component {
  render() {
    return (
      <View>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_p1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_p2')}</Text>
          <Text style={styles.paragraph}>{"\t"}{translate('injuryPainCare.Ch3_b1')}</Text>
          <Text style={styles.paragraph}>{"\t"}{translate('injuryPainCare.Ch3_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_p3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_p4')}</Text>

          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH3_TITLE_1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b5')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b6')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b7')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b8')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b9')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b10')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b11')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b12')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b13')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t1_b14')}</Text>
                      
          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH3_TITLE_2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t2_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t2_b2')}</Text>
                  
          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH3_TITLE_3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t3_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t3_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t3_b3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t3_b4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t3_b5')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t3_b6')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t3_b7')}</Text>
                  
          <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH3_TITLE_4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_p1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b1')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b2')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b3')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b4')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b5')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b6')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b7')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b8')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b9')}</Text>
          <Text style={styles.paragraph}>{translate('injuryPainCare.Ch3_t4_b10')}</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  paragraphTitle: {
    fontWeight: 'bold',
    fontSize: 20,
    alignSelf: 'stretch',
    textAlign: 'left',
    marginTop: 6,
    marginBottom: 3,
    color: colors.accent
  },
  paragraph: {
    textAlign: 'left',
    fontSize: 14,
    marginVertical: 2,
  },
  paragraphBold: {
    textAlign: 'left',
    fontSize: 14,
    fontWeight: 'bold',
    marginVertical: 2,
    marginLeft: 10
  }
});