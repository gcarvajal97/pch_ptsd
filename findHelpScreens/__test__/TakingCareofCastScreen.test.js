import React from 'react';
import TakingCareofCastScreen from '../InjuryPainCareScreens/TakingCareofCastScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<TakingCareofCastScreen/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 