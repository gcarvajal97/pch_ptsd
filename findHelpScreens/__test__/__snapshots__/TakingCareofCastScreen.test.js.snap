// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    A cast holds a broken bone in place while it heals. Casts also help to prevent or decrease muscle contractions and can help limit movement, especially after surgery.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    The outside of the cast is made from one of two types of casting materials:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    	
    - Plaster (white in color)
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    	
    - Fiberglass (comes in a variety of colors)
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Cotton and other synthetic materials are used to line the inside of the cast to make it soft and to provide padding around bony areas, such as the wrist or elbow.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Follow instructions provided by your healthcare provider (doctor or nurse practitioner) It is important that your child follow all of the special instructions given by your healthcare provider to make sure that the broken bone heals well.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Some things that your child's healthcare provider might suggest:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Keep the cast clean and dry. Do not get it wet in the shower or tub or swimming pool. Ask your doctor how you can buy a protective sleeve if you think your child might be somewhere where his cast might get wet. Check for cracks and breaks in the cast.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Check for rough edges. These rough edges can be padded to protect the skin from scratches.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Do not scratch the skin under the cast by inserting objects inside the cast â€“ this can cut the skin.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - If your child complains of itching, use a hairdryer placed on a cool setting to blow air under the cast. Never blow warm or hot air into the cast.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Do not put powders or lotions inside the cast.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Cover the cast while your child is eating to prevent food spills and crumbs from entering the cast.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Prevent small toys or objects from being put inside the cast.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Elevate the cast above the level of the heart to decrease swelling.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Encourage your child to move his or her fingers or toes to promote circulation.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Older children with body casts may need to use a bedpan or urinal in order to go to the bathroom.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Tips to keep body casts clean and dry and prevent skin irritation around the genital area include the following:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Use a diaper or sanitary napkin around the genital area to prevent leakage or splashing urine.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Place toilet paper inside the bedpan to prevent urine from splashing onto the cast or bed.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Keep the genital area as clean and dry as possible to prevent skin irritation.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Bathing and Hair Washing
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Your child may need to have sponge baths until the cast is removed.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - How to wash your child's hair: Lay your child across a bed a place a bucket on the floor to catch the water as you shampoo. You can also get 'no-rinse' shampoo at the drug store.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Make sure to ask your healthcare provider:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - How long will my child need to wear the cast?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - What are the limits to my child's activity while he wears the cast?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - What are the limits on my child's activity once you take off the cast?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - When can my child return to school?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - When can my child return to gym class? to contact sports? to biking? or to rough play?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Will my child need physical therapy?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Will this injury affect my child's growth?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    When to call your healthcare provider:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Ask your healthcare provider for specific signs and symptoms that should alert you to call them right away. They may include the following:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Your child has increasing pain and pain medicine recommended or prescribed by your healthcare provider is not helping.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - You see a change in the cast; cracking, softening, drainage from the inside.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - You smell a bad odor coming from the cast.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Your child's fingers or toes are cold, blue/gray or swollen.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Your child feels numbness or tingling in the fingers or toes.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Your child cannot move his/her fingers or toes.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - The cast is slipping off.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - There is increased swelling above or below the cast.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - The cast feels very tight.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - If you have any other concerns.
  </Text>
</View>
`;
